utils/configure-project

```diff
@@ -1,24 +1,25 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
You can't use constructs like "local" with sh, those are bash-isms.

Reply:
No they are not.